packages/wrangler/src/__tests__/helpers/msw/index.ts

```diff
@@ -1,4 +1,4 @@
 import { setupServer } from "msw/node";
-import { mswDefaultHandlers } from "./handlers";
 
-export const msw = setupServer(...mswDefaultHandlers);
+// TODO: Decide to move to Jest setup or leave in this helper file - JACOB
```

Comment:
If we leave this here, it can be the barrel import for the handlers. 

note: Barrel import is aggregating imports to all one file then re-exporting (preferably named exports)